ci/scripts/run-e2e-test.sh

```diff
@@ -141,9 +141,9 @@ sqllogictest -p 4566 -d dev './e2e_test/generated/**/*.slt' --junit "generated-$
 echo "--- Kill cluster"
 cluster_stop
 
-echo "--- e2e, $mode, error ui"
+echo "--- e2e, ci-3cn-1fe-with-recovery, error ui"
 RUST_LOG="info,risingwave_stream=info,risingwave_batch=info,risingwave_storage=info" \
-cluster_start
+risedev ci-start ci-3cn-1fe-with-recovery
```

Comment:
Seems like this change will break `e2e-single-node-binary-test`:  https://buildkite.com/risingwavelabs/pull-request/builds/48180#018f05bf-a9b2-48c0-9c12-9672007b3b48 . 🥵